src/components/application_manager/test/commands/mobile/alert_request_test.cc

```diff
@@ -235,6 +235,8 @@ TEST_F(AlertRequestTest, OnTimeout_GENERIC_ERROR) {
       (*ui_command_result)[am::strings::msg_params][am::strings::result_code]
           .asInt(),
       static_cast<int32_t>(am::mobile_api::Result::GENERIC_ERROR));
+  EXPECT_EQ((*ui_command_result)[am::strings::msg_params][am::strings::info].asString(),
```

Comment:
@pold500 It looks incorrect - according requirements info should not be empty if there is GENERIC_ERROR after timeout expired. Please correct this check